fili-core/src/main/java/com/yahoo/bard/webservice/web/util/PaginationParameters.java

```diff
@@ -90,6 +93,11 @@ private int parseParameter (String parameter, String parameterName) throws BadPa
             throw new BadPaginationException(errorMessage.format(parameterName));
         }
         try {
+            if (parameter.equals(FIRST)) {
+                return 1;
+            } else if (parameter.equals(LAST)) {
+                return LAST_PAGE;
```

Comment:
@kevinhinterlong I don't think we should officially support python-style pages just yet. I think that's a big enough change that we would want to discuss it with others first. Plus, as I think about it, I'm honestly not sure exactly how useful it would be. Really, I think we should hold off on that until some asks for the ability to easily get the "second-to-last" page.

I don't want to add an enum for the first and last page, because then we either need to have some sort of awkward either object that stores either a number or an enum, or we need to have two fields, one of which is guaranteed to be `null` at any one time.

In my opinion, the first one introduces way too much awkward infrastructure for something that is _very_ internal to the `PaginationParameters` object. Meanwhile, `-1` is a commonly used sentinel value to represent the "last" number whatever that may mean(for example, it's often used to represent infinity in configuration parameters). 

If we were passing the raw page number to other parts of the system, then the extra infrastructure would be worth it.